daemon/execdriver/driver_windows.go

```diff
@@ -0,0 +1,23 @@
+package execdriver
+
+import "github.com/docker/docker/pkg/nat"
+
+// Network settings of the container
+type Network struct {
+	Interface      *NetworkInterface `json:"interface"`
+	Mtu            int               `json:"mtu"`
+	ContainerID    string            `json:"container_id"` // id of the container to join network.
+	NamespacePath  string            `json:"namespace_path"`
```

Comment:
Since you are going to a different version of `Network` struct for windows why do you need `NamespacePath` in it. It does not make any sense in windows context.